ipaclient/install/client.py

```diff
@@ -823,7 +823,10 @@ def configure_certmonger(
 
     cmonger = services.knownservices.certmonger
     try:
-        cmonger.enable()
+        if not cmonger.is_enabled():
```

Comment:
What's the purpose of this? `cmonger.enable()` won't raise so you don't have to check for enablement.

Reply:
Good to know, I'll remove the check.